src/Documents/Generic/FELAddendas.php

```diff
@@ -0,0 +1,32 @@
+<?php 
+namespace SchoolAid\FEL\Documents\Generic;
+
+use SchoolAid\FEL\Contracts\GeneratesXML;
+use SchoolAid\FEL\Traits\HasXML;
+use SchoolAid\FEL\Enum\AddendaXML;
+
+class FELAddendas implements GeneratesXML {
+    use HasXML;
+    public function __construct(
+        private array $addendas
+    ) {}
+
+
+    public function asXML(): string
+    {
+        $addendasSubElements = [];
+        $count = 0;
+        foreach ($this->addendas as $addenda) {
+            $subElements = [
+                AddendaXML::Name->value => $addenda->getName(),
+                AddendaXML::Value->value => $addenda->getValue()
+            ];
+
+            $addendasSubElements[AddendaXML::Tag->value . (++$count)] = $subElements;
+        }
```

Comment:
Update foreach, also make this change on phrases
```php
foreach($this->addendas as $index => $addenda) {
    ...
    $addendasSubElements[AddendaXML::Tag->value . ($index + 1)] = $subElements;
}
```

Reply:
This is an old version. I've upadated this branch based on the PHP_EXAMPLE, this counter is not neccessary, same for phrases, because the tag elements always been the same name "Addenda" or "Phrase" but the value inside the element changes